LayoutTests/pointerevents/ios/touch-action-region-clip-and-transform-expected.txt

```diff
@@ -84,9 +84,7 @@
           (contentsOpaque 1)
           (drawsContent 1)
           (event region
-            (rect (0,0) width=200 height=150)
-            (rect (0,150) width=350 height=50)
```

Comment:
The previous expectation is clearly wrong I believe. width=350 but the containing graphics layer only had a bounds of 200x200